pom.xml

```diff
@@ -36,6 +35,7 @@
         <module>ssuseradd</module>
         <module>graphite</module>
         <module>es-repo</module>
+        <module>phantomjs</module>
```

Comment:
`build-rpms` profile is usually for Linux.  Please add phantomjs to `build-rpms-local` profile also so that it covers Mac as well.